src/Message.ts

```diff
@@ -279,6 +279,13 @@ export class DecodedMessage {
     this.content = content
     this.contentTopic = contentTopic
   }
+}
+
+export class DecodedMessage extends DecodedMessageBase {
+  // @returns true if the message sender is the same as the client which received the message.
+  isOutboundMessage(): boolean {
```

Comment:
Really wish this name was better. Proposed alternatives 'senderIsMe'. 

Thoughts suggestions?

Reply:
100% agree.

Reply:
My initial thought was that I agree that I prefer `isOutboundMessage` but I changed my mind as I typed this comment :).  My new opinion is that "outbound" sort of couples the name to the context in which the method is called. For example, if I have a React component that's just listening to a messages stream and rendering them, I think my mental model of the messages would be that they're all _inbound_ but some of them were _sent by me_.

So all that said my new thought is that maybe this helper should also just be called `clientAddress` and instead of returning the `boolean` it could just return `this.conversation.clientAddress` directly.

Reply:
I wouldn't object to having `clientAddress` on Messages as well, it doesn't seem much more out of place than it is on the Conversation. If we did want to preserve the conditional as well, maybe spelling out what it does could remove the contextual dependency somewhat, `senderIsTheClient`  maybe?

Reply:
> If the address is ultimately what is wanted, I am inclined to skip the helper function all together.

I think I am also inclined in that direction for what it's worth.

Reply:
Awesome. Sounds like adding `conversation.clientAddress` eases most of the developer friction. We can revisit the helper-function on `DecodedMessage` in the future if there is demand.